index.html

```diff
@@ -3104,6 +3104,280 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
+    To prevent this, JSON-LD 1.1 allows a context to be sealed.
+    </p>
+  <p>A <dfn>sealed context</dfn> is a context with a member <code>@sealed</code> set to <code>true</code>.
+    It prevents further contexts to override its term definitions.
+    </p>
+
+
+  <aside class="example ds-selector-tabs changed"
+    title="A sealed @context prevents its term definitions to be overridden">
+    <div class="selectors">
+        <button class="selected" data-selects="original">Original</button>
+        <button data-selects="expanded">Expanded</button>
+        <button data-selects="statements">Statements</button>
+        <button data-selects="turtle">Turtle</button>
+        <a class="playground" target="_blank"></a>
+    </div>
+    <pre class="original selected nohighlight" data-transform="updateExample">
+      <!--
+      {
+        "@context": [
+          {
+            ****"@version": 1.1****,
+            ****"@sealed": true****,
+            "Person": "http://schema.org/Person",
+            "name": "http://schema.org/name",
+            "knows": "http://schema.org/knows"
+          },
+          {
+            ****"Person": "this_attempt_to_override_Person_will_fail"****
+          }
+        ],
+        "@type": "Person",
+        "name": "Manu Sporny",
+        "knows": {
+          "@context": {
+            ****"name": "this_attempt_to_override_name_will_also_fail"****
+          },
+          "name": "Gregg Kellogg"
+        }
+      }
+      -->
+    </pre>
+    <pre class="expanded nohighlight"
+         data-transform="updateExample"
+         data-result-for="A sealed @context prevents its term definitions to be overridden">
+    <!--
+    [{
+      "@type": ["http://schema.org/Person"],
+      "http://schema.org/knows": [
+        {
+          "http://schema.org/name": [{"@value": "Gregg Kellogg"}],
+        }
+      ],
+      "http://schema.org/name": [{"@value": "Manu Sporny"}]
+    }]
+    -->
+    </pre>
+    <table class="statements"
+           data-result-for="A sealed @context prevents its term definitions to be overridden"
+           data-to-rdf>
+      <thead><tr><th>Subject</th><th>Property</th><th>Value</th></tr></thead>
+      <tbody>
+        <tr><td>_:b0</td><td>rdf:type</td><td>schema:Person</td></tr>
+        <tr><td>_:b0</td><td>schema:name</td><td>Manu Sporny</td></tr>
+        <tr><td>_:b0</td><td>schema:knows</td><td>_:b1</td></tr>
+        <tr><td>_:b1</td><td>schema:name</td><td>Gregg Kellogg</td></tr>
+      </tbody>
+    </table>
+    <pre class="turtle"
+         data-content-type="text/turtle"
+         data-result-for="A sealed @context prevents its term definitions to be overridden"
+         data-transform="updateExample"
+         data-to-rdf>
+      <!--
+      @prefix schema: <http://schema.org/> .
+
+      [
+        a schema:Person;
+        schema:name "Manu Sporny";
+        schema:knows [
+          shema:name "Gregg Kellogg"
+        ]
+      ] .
+      -->
+    </pre>
+  </aside>
```

Comment:
Sorry, I forgot to run rake... However, it was to be expected that the tests involving `@sealed` would fail, ad it is not yet implemented. What is the best practice here? Mark the tests with `data-ignore` for the moment.

Reply:
It’s fine for them to fail right now. Once we have consensus to merge, I can merge my own code to implement it and update the Gemfile.lock so that the examples will pass.